helper/TypeMapping.js

```diff
@@ -1,6 +1,9 @@
 const _ = require('lodash');
 const elasticsearch = require('elasticsearch');
 
+// a list of the canonical sources included in the default Pelias configuration
+const CANONICAL_SOURCES = ['whosonfirst', 'openstreetmap', 'openaddresses', 'geonames'];
```

Comment:
Let's move this to `pelias.json` to avoid more hardcoded lists of sources and layers (see https://github.com/pelias/api/issues/1161)

Reply:
Hmm.. the problem with moving it to somewhere it can be edited is that it's no longer canonical.

The whole idea with this is to be able to know which things differ from what we expect vs. what's configured.

Reply:
Yeah, but it's also something that should be located centrally, I'm sure there's code in all our repos that needs this list. The idea is no one would change it, but that's not currently enforceable, unless we added it as _code_ to pelias-config.

Reply:
Resolved in https://github.com/pelias/config/pull/103